flake.nix

```diff
@@ -1,17 +1,57 @@
 {
   description = "Complete Open Source and Modular solution for MMO";
 
-  inputs = {
-    flake-parts.url = "github:hercules-ci/flake-parts";
-    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
-  };
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
 
-  outputs = inputs@{ flake-parts, ... }:
-    flake-parts.lib.mkFlake { inherit inputs; } {
-      systems =
-        [ "x86_64-linux" "aarch64-linux" "aarch64-darwin" "x86_64-darwin" ];
-      perSystem = { config, self', inputs', pkgs, system, ... }: {
-        devShells = {
+  outputs =
+    { self, nixpkgs, ... }@attrs:
+    {
+      packages.x86_64-linux =
+        let
+          pkgs = import nixpkgs { system = "x86_64-linux"; };
+
+          mkModulePackage =
+            owner: repo: rev: hash:
+            pkgs.stdenv.mkDerivation {
+              name = repo;
+              src = pkgs.fetchFromGitHub {
+                inherit
+                  owner
+                  repo
+                  rev
+                  hash
+                  ;
+              };
+
+              dontBuild = true;
+              dontConfigure = true;
+              installPhase = ''
+                mkdir -p $out/${repo}
+                cp -R $src/* $out/${repo}
+              '';
+            };
+        in
+        rec {
+          default = server;
+          serverWithModules = server.override {
+            modules = [
+              (mkModulePackage "azerothcore" "mod-autobalance" "9048d394b4ec10cf6cb39e9f7becf8fd3ebcd123"
```

Comment:
imo mod-autobalance shouldn't be in here for the default flake

Reply:
May be good to rename it to "serverWithEluna" after removing mod-autobalance (or simply commenting autobalance out)

Reply:
It was more about showing an example on how to add a module but I like the idea is having one with Eluna since it's so common.